### PR TITLE
fixed diff highlighting in docs tutorial

### PR DESCRIPTION
**Pull request by SCasarotto**

### CODE REVIEW

File: contributors.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 - mcansh
+- SCasarotto
```
Comment (MichaelDeBoey) [inline contributors.yml:2]: Should be lowercase for some reason 🤷‍♂️

```suggestion
- scasarotto

```

**Comment by SCasarotto** [comment]
@MichaelDeBoey Thanks for the help there.

File: contributors.yml (hunk 1 above)
Comment (mcansh) [reply contributors.yml]: ah shoot, i fixed this in the inverse 😂

**Review by kentcdodds — COMMENTED** [review]
Thanks! We've got other tutorials that are going to replace this one though. Appreciate you taking the time though!

**Pull request closed without merging**